Javascript/Ez Namey /namey.js

```diff
@@ -0,0 +1,43 @@
+let firstWords = [
```

Comment:
Nearly all `let` in this file can be set as `const`.

Reply:
***Changed.***